js/common/charts.d.ts

```diff
@@ -277,3 +281,57 @@ export type WordWrap = 'normal' | 'breakWord' | 'none';
  * @namespace DevExpress.common.charts
  */
 export type ZoomPanAction = 'zoom' | 'pan';
+
+/**
+ * @docid
+ * @type object
+ * @public
+ * @namespace DevExpress.common.charts
+ */
+export type ChartsColor = {
+    /**
+     * @docid
+     * @default undefined
+     * @public
+     */
+    base?: string;
+    /**
+     * @docid
+     * @default undefined
+     * @public
+     */
+    fillId?: string;
+};
+
+/**
+ * @docid viz.registerGradient
+ * @publicName registerGradient(type, options)
+ * @static
+ * @public
+ */
+export function registerGradient(type: string, options: { rotationAngle?: number; colors: Array<GradientColor> }): string;
+/**
+ * @docid viz.registerPattern
+ * @publicName registerPattern(options)
+ * @static
+ * @public
+ */
+export function registerPattern(options: { width: number | string; height: number | string; template: template }): string;
+/**
+ * @namespace DevExpress.viz
```

Comment:
All things here should have the same namespace
```suggestion
 * @namespace DevExpress.common.charts
```